src/basic/process-util.c

```diff
@@ -806,6 +806,13 @@ void sigkill_waitp(pid_t *pid) {
         sigkill_wait(*pid);
 }
 
+void sigterm_wait(pid_t pid) {
```

Comment:
Where is this new function used?

Reply:
whoops. Meant to use it in bus_kill_exec, which I just fixed.